incubator/virtualcluster/pkg/controller/pki/pki.go

```diff
@@ -110,17 +110,19 @@ func NewAPIServerKubeletClientCertAndKey(ca *CrtKeyPair) (*x509.Certificate, *rs
 }
 
 // NewEtcdServerCrtAndKey creates new crt-key pair using ca for etcd
-func NewEtcdServerCrtAndKey(ca *CrtKeyPair, etcdDomain string) (*CrtKeyPair, error) {
-
+func NewEtcdServerCrtAndKey(ca *CrtKeyPair, etcdDomains []string) (*CrtKeyPair, error) {
 	// create AltNames with defaults DNSNames/IPs
+	// log.Println("adding etcd domains into cert ", "domains ", etcdDomains)
```

Comment:
Can we remove this uncomment line? or change it to a debug log.

Reply:
useless comment removed